src/main/java/uk/gov/digital/ho/hocs/casework/migration/api/MigrationCaseDataService.java

```diff
@@ -152,4 +142,39 @@ public void updatePrimaryCorrespondent(UUID caseUUID, UUID stageUUID, UUID prima
         log.info("Updated Primary Correspondent for Migrated Case: {} Correspondent: {}", caseUUID, primaryCorrespondentUUID,
             value(EVENT, PRIMARY_CORRESPONDENT_UPDATED));
     }
+
+    void createAdditionalCorrespondent(List<MigrationComplaintCorrespondent> additionalCorrespondents, UUID caseUUID, UUID stageUUID) {
+        for (MigrationComplaintCorrespondent additionalCorrespondent : additionalCorrespondents) {
+            log.debug("Creating Additional Correspondent of Type: {} for Migrated Case: {}", additionalCorrespondent.getCorrespondentType(), caseUUID);
+            Correspondent correspondent = getCorrespondent(additionalCorrespondent, caseUUID);
+
+            try {
+                correspondentRepository.save(correspondent);
+                auditClient.createCorrespondentAudit(correspondent);
+
+            } catch(DataIntegrityViolationException e) {
+                throw new ApplicationExceptions.EntityCreationException(
+                    String.format("Failed to create correspondent %s for Migrated Case: %s", correspondent.getUuid(), caseUUID),
+                    CORRESPONDENT_CREATE_FAILURE, e);
+            }
+            log.info("Created Correspondent: {} for Migrated Case: {}", correspondent.getUuid(), caseUUID,
+                value(EVENT, CORRESPONDENT_CREATED));
+        }
+    }
+
+    private Correspondent getCorrespondent(MigrationComplaintCorrespondent primaryCorrespondent, UUID caseUUID) {
```

Comment:
`primaryCorrespondent` is probably the wrong name here as its used within additional correspondents too.